musicbrainz/csv/get_genre.py

```diff
@@ -0,0 +1,62 @@
+"""
+Script for retrieving the "genre" dumps since they are not available online.
+"""
+
+import time
+import requests
+from requests.exceptions import HTTPError
+import pandas as pd
+from bs4 import BeautifulSoup
+
+URL = "https://musicbrainz.org/ws/2/genre/all"
+PARAMS = dict(fmt="json", limit="50")
+
+data = []
+max_records = dict(requests.get(url=URL, params=PARAMS, timeout=500).json())[
+    "genre-count"
+]
+
+for i in range(0, max_records):
+    PARAMS["offset"] = i
```

Comment:
This looks like your `offset` parameter is incrementing by 1 but your `limit` is 50...won't you request the same genre over and over again?

Reply:
If I my offset increase 50 each time and my limit is 50, then is it making 50 API calls or is it making 1 API call?

Reply:
1 request.get call = 1 API call?

I'm not sure I understand your question.

Reply:
1 request.get API call that requests 50 record at once
versus
1 request.get API call that only requests 1 record

Reply:
OK... my only question is is this duplicating the genres you are getting back?

Reply:
Right now it works fine, I changed i back to jumping in 50s.